tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -85,11 +85,6 @@ def _get_vmware_datastore_summary_string(data_store_name=VMWARE_CONSTANTS['datas
     return f'{data_store_name} (free: {free_space}, prov: {prov}, total: {capacity})'
 
 
-@pytest.fixture(scope='module')
-def module_org():
-    return entities.Organization().create()
-
-
 @pytest.fixture(scope='module')
 def module_vmware_settings():
     ret = dict(
```

Comment:
Can we remove the use of this fixture as well and use settings object directly, same as CLI module for consistency?

Reply:
Removed